### PR TITLE
src/rgw: Issue #50138 fix

### PR DESCRIPTION
**Pull request by TRYTOBE8TME**
This PR fixes https://tracker.ceph.com/issues/50138

Signed-off-by: Kalpesh Pandya <kapandya@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/test/rgw/bucket_notification/test_bn.py [hunk 1]
```diff
@@ -1006,74 +1006,77 @@ def test_ps_s3_notification_push_kafka_on_master():
...
                        {'Id': notification_name + '_2', 'TopicArn': topic_arn2,
                          'Events': []
                        }]
 
-    s3_notification_conf = PSNotificationS3(conn, bucket_name, topic_conf_list)
-    response, status = s3_notification_conf.set_config()
-    assert_equal(status/100, 2)
+        s3_notification_conf = PSNotificationS3(conn, bucket_name, topic_conf_list)
+        response, status = s3_notification_conf.set_config()
+        assert_equal(status/100, 5)
```
Comment (yuvalif) [inline src/test/rgw/bucket_notification/test_bn.py:1034]: why did you change from 200 to 500?

**Comment by TRYTOBE8TME** [comment]
Teuthology run: http://pulpito.front.sepia.ceph.com/kapandya-2021-04-05_11:01:38-rgw:notifications-master-distro-basic-smithi/

File: src/test/rgw/bucket_notification/test_bn.py (hunk 1 above)
Comment (TRYTOBE8TME) [reply src/test/rgw/bucket_notification/test_bn.py]: Addressed.

**Review by yuvalif — APPROVED** [review]

**Comment by yuvalif** [comment]
please change the commit headline to:
"rgw/test: stop kafka consumer if tests fails"
please read: https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#commit-title
commit title has to be descriptive (not "fix xxx")
